draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1363,10 +1371,11 @@ uCDN-A                         dCDN-B                         dCDN-C
    <section anchor="generic-spec-object" title="Generic Spec Object">
    <t>
 	A trigger resource, as defined in <xref target="ci-trigger-resource"/>, includes an array of trigger spec objects. 
-	Each trigger spec object contains properties that are used as trigger target selection directives for dCDN when processing the trigger. 
-	e.g. content URLs or metadata URI patterns. Each such trigger spec is a specialization of a CDNI GenericTriggerSpec object. 
-	The GenericTriggerSpec object abstracts the basic information required for trigger distribution from the specifics of any given property
-	(i.e., property semantics, enforcement options, etc.). 
+	Each trigger spec object contains properties that are used as trigger target selection directives for dCDN when 
+	processing the trigger.  e.g., content URLs or metadata URI patterns. Each such trigger spec is a specialization 
```

Comment:
The period after trigger should likely be a comma so that "content URLs or metadata URI patterns isn't a sentence fragment.